data/mods/Magiclysm/enchanted/enchanted_wands.json

```diff
@@ -0,0 +1,362 @@
+[
+  {
+    "id": "wand_magic_missile_minor",
+    "name": "wand of magic missiles",
+    "plural": "wands of magic missile",
+    "description": "A classic, this wand shoots magic missiles without fail.  A more skilled Magus could cast them more efficiently, though.",
+    "type": "TOOL",
+    "weight": 230,
+    "volume": 2,
+    "price": 10000,
+    "bashing": 3,
+    "to_hit": 2,
+    "material": [ "wood" ],
+    "symbol": "|",
+    "color": "brown",
+    "charges_per_use": 1,
+    "ammo": "crystallized_mana",
+    "use_action": { "type": "cast_spell", "spell_id": "magic_missile", "no_fail": true, "level": 5 },
+    "magazines": [ [ "crystallized_mana", [ "small_mana_crystal" ] ] ],
+    "magazine_well": 1
+  },
+  {
+    "id": "wand_magic_missile_lesser",
```

Comment:
this is a carbon copy of wand_magic_missile. what's the point of having two wands that have different ids but the exact same stats?